pkg/R/DataFrame.R

```diff
@@ -276,3 +298,37 @@ setMethod("foreachPartition",
             rddIn <- toRDD(rdd)
             foreachPartition(rddIn, func)
           })
+
+
+############################## GroupedData ########################################
+
+setClass("GroupedData",
+         slots = list(env = "environment",
+                      sgd = "jobj"))
+
+setMethod("initialize", "GroupedData", function(.Object, sgd) {
+  .Object@env <- new.env()
+  .Object@sgd <- sgd
```

Comment:
Is there a `cache` or `persist` method for GroupedData? If so, we probably want to add an `isCached` flag in `.Object@env`